### DISCUSSION TITLE
Strict slashes on websocket

### DISCUSSION THREAD

**Discussion by MrNom4ik** [fastapi/fastapi#9575] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI, WebSocket

app = FastAPI(debug=True)


@app.websocket("/ws")
async def websocket_router_endpoint(websocket: WebSocket):
    await websocket.accept()
    await websocket.send_text("Hello from FastAPI!")
    await websocket.close()
```

### Description

If I try to connect to `/ws` everything is fine, but if I try to connect to `/ws/` I get a 403 error.
How can connect in two ways at once?

### Operating System

Linux

### FastAPI Version

0.95.2

### Python Version

3.9.0

### Additional Context

Console output:
```
INFO:     Uvicorn running on http://127.0.0.1:8900 (Press CTRL+C to quit)
INFO:     ('127.0.0.1', 51909) - "WebSocket /ws" [accepted]
INFO:     connection open
INFO:     connection closed
INFO:     ('127.0.0.1', 51913) - "WebSocket /ws/" 403
INFO:     connection failed (403 Forbidden)
INFO:     connection closed
```

**Comment by silverman63** [comment]
I did some searching, and it seems like this issue [goes in and out of being resolved in Starlette](https://github.com/encode/starlette/issues/869) for regular routing, let alone websockets.

The best solution I could come up is just an explicit additional mapping
```python
app.add_api_websocket_route("/ws/", websocket_router_endpoint)
```

**Comment by YuriiMotov** [comment]
You can add two route decorators like it's shown below:

```python
from fastapi import FastAPI, WebSocket
from fastapi.testclient import TestClient
import pytest

app = FastAPI(debug=True, redirect_slashes=True)


@app.websocket("/ws/")
@app.websocket("/ws")
async def websocket_router_endpoint(websocket: WebSocket):
    await websocket.accept()
    await websocket.send_text("Hello from FastAPI!")
    await websocket.close()


@pytest.mark.parametrize("path", ["/ws", "/ws/"])
def test(path: str):
    client = TestClient(app)

    with client.websocket_connect(path) as ws:
        ws.receive_text()

```
